test/e2e/framework/metrics_util.go

```diff
@@ -53,7 +53,7 @@ const (
 
 	// We set a higher threshold for list apicalls as they can take more time when
 	// the list is really big. For eg. list nodes in a 5000-node cluster.
-	apiListCallLatencyThreshold time.Duration = 2 * time.Second
+	apiListCallLatencyThreshold time.Duration = 5 * time.Second
```

Comment:
Besides, as a user of the SLO, it will be an incentive for me to keep the count of my pods very close to Y, but < Y. So that I'm squeezing maximum guarantees out of the SLO.